package.json

```diff
@@ -51,8 +51,5 @@
       "prettier --write"
     ]
   },
-  "packageManager": "yarn@3.4.1",
-  "dependencies": {
-    "@backstage/plugin-sonarqube": "^0.6.5"
```

Comment:
The SonarQube plugin won't work if it's not installed.
I will check if it can be moved to the frontend dependencies but from the documentation, it's in this package.json file.

Reply:
The SonarQube plugin does not need to be installed in the root `package.json`. It is already installed under the `packages/app` `package.json`. For future reference, the only packages that need to be installed on the root level are workspace wide dependencies like `turbo`.